cypress.config.js

```diff
@@ -15,4 +17,11 @@ module.exports = defineConfig({
     },
     baseUrl: 'http://localhost:3000',
   },
+
```

Comment:
Was this added by mistake? We're not using Cypress to do component tests as we have Jest for that

Reply:
Corrected.